shared/chat/conversation/header/index.native.js

```diff
@@ -7,8 +7,8 @@ import type {Props} from '.'
 
 const ConversationHeader = ({muted, onBack, onOpenFolder, onShowProfile, onToggleSidePanel, sidePanelOpen, users}: Props) => (
   <Box style={containerStyle}>
-    <BackButton title={null} onClick={onBack} iconStyle={{color: globalColors.blue}} textStyle={{color: globalColors.blue}} style={{flexShrink: 0}} />
-    <Box style={{...globalStyles.flexBoxRow, justifyContent: 'center', flex: 1, marginTop: 2}}>
+    <BackButton title={null} onClick={onBack} iconStyle={{color: globalColors.blue}} textStyle={{color: globalColors.blue}} style={{flexShrink: 0, padding: globalMargins.tiny}} />
```

Comment:
we don't have to make this change in the short term but ideally we'd have just the headerhoc and you'd be able to give it children that'd go in there instead of a title.